README.md

```diff
@@ -723,5 +860,73 @@ Note: This assumes you have additional components or code to allow files to actu
 ```
 
 #### `<thumbnail />`
+The `<Thumbnail />` component allows you to easily render Fine Uploader generated thumbnail previews for a specific submitted file. While the thumbnail generation is in progress, a SVG "waiting" graphic will render. Of the thumbnail generation succeeds, the "waiting" graphic will be removed from the DOM and replaced with a <canvas> element containing the thumbnail preview. If thumbnail generation fails, a "not available" SVG graphic will be rendered instead.
+
+##### Properties
+- `customResizer(resizeInfo)` - An optional function that allows you to use a custom/3rd-library to resize thumbnail images. Documented further in [Fine Uploader's `drawThumbnail` API method documentation](https://docs.fineuploader.com/api/methods.html#drawThumbnail). See the second code example below for details.
+
+- `fromServer` - Specify whether the current file was set from [initial file](https://docs.fineuploader.com/branch/master/features/session.html)
+
+- `id` - The Fine Uploader ID of the submitted file. (required)
+
+- `maxSize` - Maps directly to the [`maxSize` parameter](http://docs.fineuploader.com/branch/master/api/methods.html#drawThumbnail) of the Fine Uploader `drawThumbnail` API method. If not supplied a default value is used, which is exported as a named constant.
+
+- `uploader` - A Fine Uploader [wrapper class](#wrapper-classes). (required)
+
+Suppose you wanted to render a thumbnail for each file as new files are submitted to Fine Uploader. Your React component may look like this:
+
+Note: This assumes you have additional components or code to allow files to actually be submitted to Fine Uploader.
+
+##### Slots
+- `notAvailablePlaceholder` - A custom element to display if the thumbnail is not available.
+
+- `waitingPlaceholder` - A custom element to display while waiting for the thumbnail.
+
+Suppose you wanted to render a thumbnail for each file as new files are submitted to Fine Uploader. Your Vue component may look like this:
+
+Note: This assumes you have additional components or code to allow files to actually be submitted to Fine Uploader.
+
+```js
+<template>
+  <div v-for="file in state.submittedFiles">
+    <thumbnail :id="file" :uploader="uploader" />
+    <cancel-button :id="file" :uploader="uploader" />
+    <delete-button :id="file" :uploader="uploader" />
```

Comment:
I would remove the `delete` and the `cancel-button`, they're not needed in the `thumbnail` part of the README.

Reply:
Great catch! Thanks